registration/views.py

```diff
@@ -47,6 +48,8 @@
 SESSION_VARIANT_ID = 'variant_id'
 SESSION_REGISTRATION_UUID = 'registration_uuid'
 SESSION_FIRST_NAME = 'first_name'
+SESSION_LICENSE_ID_FORMATS = 'license_id_formats'
+SESSION_LICENSE_ID_REGEXES = 'license_id_regexes'
```

Comment:
@mabuaisha I think that we don't need a key for regexes anymore, since we are not saving it in session.